reign.py

```diff
@@ -116,15 +121,27 @@ def battle(self):
         """
 
         # Choose ∑
-        empire = random.choice(self.obj.Empire.values.tolist())
-        empire_neighbours = self.obj.query(f'Empire == "{empire}"').iloc[0].empire_neighbours
+
+        empire_list = self.obj.Empire.values.tolist()
+        un_empire, empire_weight = np.unique(empire_list, return_counts=True)
```

Comment:
Should `un_empire` be `unique_empires` and `empire_weight` `empire_weights`?

Reply:
yes